packages/ui-top-nav-bar/src/TopNavBar/TopNavBarLayout/DesktopLayout/index.tsx

```diff
@@ -129,7 +149,24 @@ class TopNavBarDesktopLayout extends Component<TopNavBarDesktopLayoutProps> {
           <div css={styles?.brandContainer}>{renderBrand}</div>
         )}
 
-        <div css={styles?.menuItemsContainer}>{renderMenuItems}</div>
+        {this.hasMenuItemsBlock && (
+          <div css={styles?.menuItemsContainer}>{renderMenuItems}</div>
+        )}
+
+        {!(this.hasBrandBlock || this.hasMenuItemsBlock) &&
+          this.hasBreadcrumbBlock && (
```

Comment:
small suggestion: maybe `hasBreadcumbBlock` could be renamed to `shouldRenderBreadcumb` or something similar and this logic moved there to be more readable and to have the display logic in one place